lib/poller.js

```diff
@@ -23,27 +23,47 @@ class Poller {
    * @param {Object} kubeClient - Client for interacting with kubernetes cluster.
    * @param {Object} logger - Logger for logging stuff.
    * @param {string} namespace - Kubernetes namespace.
-   * @param {SecretDescriptor} secretDescriptor - Kubernetes secret descriptor.
+   * @param {Object} customResourceManifest - CRD manifest
+   * @param {Object} externalSecret - ExternalSecret custom resource object.
```

Comment:
comment nit: "ExternalSecret manifest" instead of "ExternalSecret custom resource object".  When I see "custom resource object" i think CRD / Custom Resource Definition object.

Reply:
Thanks it didn't feel right :smile: